stable_baselines3/td3/td3.py

```diff
@@ -167,11 +167,10 @@ def learn(self,
               reset_num_timesteps: bool = True) -> OffPolicyRLModel:
 
         callback = self._setup_learn(eval_env, callback, eval_freq,
-                                     n_eval_episodes, eval_log_path, reset_num_timesteps)
+                                     n_eval_episodes, eval_log_path, reset_num_timesteps,
+                                     self.verbose, self.tensorboard_log, tb_log_name)
```

Comment:
why do you pass `self.verbose`? You have access to it inside the class...

Reply:
Ah good catch